client/src/util/finiteExtent.ts

```diff
@@ -21,14 +24,17 @@ function finiteExtent(tarr: any) {
       break;
     }
   }
-  for (; i < tarr.length; i += 1) {
-    const val = tarr[i];
-    if (Number.isFinite(val)) {
-      if (min > val) min = val;
-      if (max < val) max = val;
+  if (min !== undefined && max !== undefined) {
```

Comment:
@bkmartinjr I see this is runtime code modification, does this look good to you 😆 ? 
Is there a case that we want to enter the second loop even if only either `min` or `max` is `undefined`?

Reply:
the test is equivalent either way (&& or ||) as there is no way for only one to be undefined.  Either _both_ are undefined, or neither are undefined.

@seve - if you want to micro-optimize (perhaps even femto-optimize :-), you could change it to an ||.  But the logic is the same in any case.